packages/catalog-realm/product-requirement-document.gts

```diff
@@ -13,8 +13,7 @@ import {
 import { Button } from '@cardstack/boxel-ui/components';
 import { ImagePlaceholder } from '@cardstack/boxel-ui/icons';
 import { bool, cn, not } from '@cardstack/boxel-ui/helpers';
-import { on } from '@ember/modifier';
-import { action } from '@ember/object';
+import { on } from '@ember/modifier'; // TODO: why is this a type error???
```

Comment:
hmmm, I vaguely recall doing something special fro base realm and expreiments realm to get the types correct, we need to do that here too